cv32e40s/tests/cfg/default.yaml

```diff
@@ -4,176 +4,14 @@ compile_flags: >
-    --override cpu/mask_pmpaddr20=0x00000000
-    --override cpu/mask_pmpaddr21=0x00000000
-    --override cpu/mask_pmpaddr22=0x00000000
-    --override cpu/mask_pmpaddr23=0x00000000
-    --override cpu/mask_pmpaddr24=0x00000000
-    --override cpu/mask_pmpaddr25=0x00000000
-    --override cpu/mask_pmpaddr26=0x00000000
-    --override cpu/mask_pmpaddr27=0x00000000
-    --override cpu/mask_pmpaddr28=0x00000000
-    --override cpu/mask_pmpaddr29=0x00000000
-    --override cpu/mask_pmpaddr30=0x00000000
-    --override cpu/mask_pmpaddr31=0x00000000
-    --override cpu/mask_pmpaddr32=0x00000000
-    --override cpu/mask_pmpaddr33=0x00000000
-    --override cpu/mask_pmpaddr34=0x00000000
-    --override cpu/mask_pmpaddr35=0x00000000
-    --override cpu/mask_pmpaddr36=0x00000000
-    --override cpu/mask_pmpaddr37=0x00000000
-    --override cpu/mask_pmpaddr38=0x00000000
-    --override cpu/mask_pmpaddr39=0x00000000
-    --override cpu/mask_pmpaddr40=0x00000000
-    --override cpu/mask_pmpaddr41=0x00000000
-    --override cpu/mask_pmpaddr42=0x00000000
-    --override cpu/mask_pmpaddr43=0x00000000
-    --override cpu/mask_pmpaddr44=0x00000000
-    --override cpu/mask_pmpaddr45=0x00000000
-    --override cpu/mask_pmpaddr46=0x00000000
-    --override cpu/mask_pmpaddr47=0x00000000
-    --override cpu/mask_pmpaddr48=0x00000000
-    --override cpu/mask_pmpaddr49=0x00000000
-    --override cpu/mask_pmpaddr50=0x00000000
-    --override cpu/mask_pmpaddr51=0x00000000
-    --override cpu/mask_pmpaddr52=0x00000000
-    --override cpu/mask_pmpaddr53=0x00000000
-    --override cpu/mask_pmpaddr54=0x00000000
-    --override cpu/mask_pmpaddr55=0x00000000
-    --override cpu/mask_pmpaddr56=0x00000000
-    --override cpu/mask_pmpaddr57=0x00000000
-    --override cpu/mask_pmpaddr58=0x00000000
-    --override cpu/mask_pmpaddr59=0x00000000
-    --override cpu/mask_pmpaddr60=0x00000000
-    --override cpu/mask_pmpaddr61=0x00000000
-    --override cpu/mask_pmpaddr62=0x00000000
-    --override cpu/mask_pmpaddr63=0x00000000
-    --override cpu/mask_pmpcfg0=0x00000000
-    --override cpu/mask_pmpcfg1=0x00000000
-    --override cpu/mask_pmpcfg2=0x00000000
-    --override cpu/mask_pmpcfg3=0x00000000
-    --override cpu/mask_pmpcfg4=0x00000000
-    --override cpu/mask_pmpcfg5=0x00000000
-    --override cpu/mask_pmpcfg6=0x00000000
-    --override cpu/mask_pmpcfg7=0x00000000
-    --override cpu/mask_pmpcfg8=0x00000000
-    --override cpu/mask_pmpcfg9=0x00000000
-    --override cpu/mask_pmpcfg10=0x00000000
-    --override cpu/mask_pmpcfg11=0x00000000
-    --override cpu/mask_pmpcfg12=0x00000000
-    --override cpu/mask_pmpcfg13=0x00000000
-    --override cpu/mask_pmpcfg14=0x00000000
-    --override cpu/mask_pmpcfg15=0x00000000
-    #--trace
-    #--tracechange
+    --override cpu/hpmcounter_undefined=T
+    --override cpu/PMP_registers=64
+    --override cpu/PMP_grain=0
     #--showoverrides
-    #--traceshowicount
-    #--monitornets
-    #--override root/showload=T
```

Comment:
These PMP-settings cannot be removed - ISS will be wrongly configured; at minimum the mask_* overrides must remain in place as the RTL has 0 active PMP, regions (The ISS must be configured with 64 RO always zero configuration registers)